src/libstd/io/mod.rs

```diff
@@ -322,6 +587,21 @@ pub trait Write {
     ///
     /// It is **not** considered an error if the entire buffer could not be
     /// written to this writer.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// use std::io::Write;
+    /// use std::fs::OpenOptions;
+    ///
+    /// # fn foo() -> std::io::Result<()> {
+    /// let mut buffer = try!(OpenOptions::new().write(true).open("foo.txt"));
+    ///
+    /// buffer.write(b"some bytes").unwrap();
+    /// buffer.flush().unwrap();
```

Comment:
I may have lost this comment in the emails (thanks github), but could the `flush()` in this and all below examples be omitted? I think this may only need to be showed off on the `flush()` method itself.